packages/hoc-input/src/formInput.js

```diff
@@ -135,6 +168,19 @@ const formInput = WrappedComponent => {
       this.setState(nextState);
     }
 
+    handleClearClick = () => {
+      this.setState({ value: "" });
+      this.props.onChange({
+        type: "change",
+        value: "",
+        target: { value: "" }
```

Comment:
this should be fine, but passing target: this.inputRef should also work, right?

Reply:
Actually, no.
Passing `this.inputRef` as target causes the browser to freeze. 👀